src/main/java/com/spotinst/sdkjava/example/elastigroup/aws/ElastigroupUsageExample.java

```diff
@@ -180,6 +180,29 @@ public static void main(String[] args) throws IOException {
         System.out.println("----------Get Suggested Instance Types--------------");
         List<GetInstanceTypesResponse> getSuggestedInstanceTypes = getSuggestedInstanceTypes(elastigroupClient);
 
+        //Initiate Roll
+        System.out.println("----------Initiate Roll--------------");
+        ElastigroupEcsClusterRollResponse initiateRollResponse = initiateClusterRoll(elastigroupClient, "group-id", 25, "comment", true, 100);
+
+        //Get cluster Roll
+        System.out.println("----------Get cluster Roll--------------");
+        ElastigroupEcsClusterRollResponse getClusterRollStatus = getClusterRoll (elastigroupClient, "group-id", "rollId");
+
+        //List cluster Rolls
+        System.out.println("----------List cluster Rolls--------------");
+        List<ElastigroupEcsClusterRollResponse> listClusterRolls = getAllClusterRolls(elastigroupClient, "group-id");
+
+        //Update cluster Roll
+        System.out.println("----------Update cluster Roll--------------");
+        ElastigroupEcsClusterRollResponse updateRollResponse = updateClusterRoll(elastigroupClient, "group-id", "STOPPED");
+
+        //Create Instance Signal
+        System.out.println("----------Create Instance Signal--------------");
+        Boolean createInstanceSignalStatus = createInstanceSignal(elastigroupClient);
```

Comment:
Should mention what signal is to be created.

Reply:
fixed